src/Domain/ApplicationTime.php

```diff
@@ -69,6 +70,10 @@ public static function setTime(int $appTime = null)
     {
         static::$appTime = DateTimeImmutable::createFromFormat('U', $appTime ?? time())
             ->setTimezone(new DateTimeZone('UTC'));
+
+        if (class_exists(Chronos::class)) {
```

Comment:
Should we do this for the [other Chronos classes too](https://book.cakephp.org/3.0/en/chronos.html#testing-aids)? I can't see us ever using the Mutable classes, but `Date` feels like it'll come in handy in the future